pvxarray/cf.py

```diff
@@ -0,0 +1,12 @@
+def get_cf_names(da):
+    """Use `cf_xarray` to get the names of the X, Y, and Z arrays."""
+    try:
+        import cf_xarray  # noqa
+
+        axes = da.cf.axes
+    except (AttributeError, ImportError):
+        raise ImportError("Please import `cf_xarray` to use CF conventions.")
+    x = axes.get("X", [None])[0]
+    y = axes.get("Y", [None])[0]
+    z = axes.get("Z", [None])[0]
```

Comment:
This just feels wrong

Reply:
I agree that is quite ugly.

Alternatives are

```
x = da.axes["X"][0] if "X" in da.cf else None
```

or in `_get_array`:
```
# if cf_xarray exists
return da.cf[key]  # this falls back to usually xarray stuff if "key" is not interpretable using cf conventions
```

Reply:
Great! thank you for the suggestions